src/main/java/core/basesyntax/service/impl/FileServiceCsvImpl.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileService;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileServiceCsvImpl implements FileService {
+    @Override
+    public List<String> readFile(String fileName) {
+        try {
```

Comment:
According to the checklist, you should use try-with-resources for classes that implement the AutoCloseable interface. In this case, Files.readAllLines() does not need it because it does not require closing resources, but make sure to apply this principle in situations where it's needed.